interface/src/app/gene/gene-network.tpl.html

```diff
@@ -2,17 +2,28 @@
   <h1 class="page-header">
     Gene Network
   </h1>
-  <autocomplete-search-panel organism="searchCtrl.organism" autocomplete="searchCtrl.autocomplete"
-    ng-if="searchCtrl.autocomplete">
+
+  <m-l-model-selector selected-m-l-model="searchCtrl.selectedMLModel">
```

Comment:
Ah I see, well especially in the light of @mhuyck's comments, I will change it to ``mlModelSelector`` and ``selectedMlModel`` to make it both easier to read and also improve the annoying tag names. Thanks!